eng/config/PublishData.json

```diff
@@ -196,28 +196,29 @@
       "version": "4.0.*",
       "packageFeeds": "default",
       "channels": [],
-      "vsBranch": "rel/d17.0",
+      "vsBranch": "main",
```

Comment:
It wasn't clear to me why the vsBranch changed from rel/d17.0 to main here. Related to #56943